pkg/services/postgresql/catalog.go

```diff
@@ -167,7 +167,7 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 				ID:             "d3f74b44-79bc-4d1e-bf7d-c247c2b851f9",
 				Name:           "azure-postgresql-9-6-dbms",
 				Description:    "Azure Database for PostgreSQL 9.6-- DBMS only",
-				ChildServiceID: "25434f16-d762-41c7-bbdd-8045d7f74ca",
+				ChildServiceID: "25434f16-d762-41c7-bbdd-8045d7f74ca6",
```

Comment:
a strange change?

Reply:
At first we considered just using V2 GUIDs so I modified them all. Then, we considered using an option to choose the GUID set. It is changed by mistake when I roll back the GUIDs... Thanks for the good catch. Will fix.

Reply:
Oh, the origin one is actually missing the last letter. This is a fix.